projects/GKv2/GEDKeeper2/GKUI/Forms/TreeChartWin.cs

```diff
@@ -109,12 +109,15 @@ public TreeChartWin(IBaseWindow baseWin, GDMIndividualRecord startPerson)
 
             fController = new TreeChartWinController(this);
             fController.Init(baseWin);
+
+            miGens9_InitialSetup();
         }
 
         protected override void Dispose(bool disposing)
         {
             if (disposing) {
-                if (components != null) components.Dispose();
+                if (components != null)
+                    components.Dispose();

```

Comment:
Before you make a commit, you need to check - so that edits that you did not do to solve the problem do not fall into it. You need to configure VS so that it does not change the formatting where you do not make changes.

Reply:
Automatic formatting changes that clutter the contents of the commit (AFC marker).